package.json

```diff
@@ -11,7 +11,7 @@
     "cookie-parser": "^1.4.5",
     "ejs": "^3.1.5",
     "express": "^4.17.1",
-    "lens": "elifesciences/lens.git#develop",
+    "lens": "github:elifesciences/lens#develop",
```

Comment:
This is interesting, I've not come across this style of dependency before. Nice to learn something new :-) That did lead me to read [the docs](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#git-urls-as-dependencies) though, and I can't see anything about the `github:` prefix when specifying a dependency. I can see it [mentioned under the repository section though](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#repository). Hence, are we sure this is correct? Might be safer to stick to...

```
elifesciences/lens#develop
```

My concern I guess is that this probably doesn't need to change, so are we sure we want to change it?

Reply:
It is the output produced automatically for me by the tooling, and now we're aware of it, I'm happy to remove the `github:` portion for now.

Reply:
Ah, I'll hazard a guess that locally you have a much newer version of Node.js and NPM than we have on the CI. I know on the CI it'll either be Node.js v6.x or v8.x depending on how the formula is setup. As such, I'd revert this back to what it was (which we know should work with the CI) and that should then safely make it through.

Reply:
Note, I say CI but actually there doesn't actually seem to be any having just checked Jenkins. No formula either checking GitHub, in which case I expect the above change might be fine actually!

Reply:
Probably yes, I reverted it back now.